### DISCUSSION TITLE
Feature Request: Dynamic Opacity and Blur Settings Based on Light/Dark Mode

### DISCUSSION THREAD

**Discussion by webel** [ghostty-org/ghostty#3773] [Feature Requests, Ideas] [answered]
Hi,

I like Ghostty so far, the only thing I've missed coming from a scripting config is the ability to set "stuff" depending on dark or light mode. 
E.g. background-opacity and background-blur-radius using the light:..., dark:... syntax, similar to how themes work. 

```
background-opacity = "light:0.15, dark:0.8"
background-blur-radius = "light:20, dark:15"
```

This would make it much easier to adjust appearance dynamically without relying on external scripts.

Thanks for Ghostty ❤️

**Comment by mitchellh** [comment, marked as answer]
I don't know the duplicate, but we can merge this into the conditional config discussion. We have the underlying system to handle this we just need to figure out the syntax.
